Makefile

```diff
@@ -89,6 +89,9 @@ build_ios_static: ## Build the static library target
 build_carthage: ## Build the latest pushed commit with Carthage
 	@./scripts/build-carthage.sh
 
+build_xcframework: ## Build as a multiplatform xcframework
```

Comment:
nit: The Make target calls it xcframework but the script just uses framework.  Maybe less likely to trip people up if we us framework for both?  I can change it on the PR to add it to CI, so long as you agree.

Reply:
Oh, it's actually building an xcframework, so best to settle on that I guess

Reply:
Yeah, I don't mind at all which one is used, I'm just likely to forget and get them wrong if they aren't the same - I'll chang eit to xc when I get to it no problem.

Reply:
Everything should be xcframework now